### ISSUE TITLE
Get value from a FC layer?

### ISSUE THREAD

**Issue by Yunrui** [dmlc/mxnet#1920]
Hi folks, 

Is there any way to get value from any specific FC layer? 

I trained a multi-layers model with Softmax as the last one. But what I really want is value of second last layer. Hoping there is a way to get value by specifying name of any layer. 

thx

**Comment by Yunrui** [comment]
I see MXPredPartialForward was added to code base, but not see where to use it.

**Comment by tqchen** [comment]
see last block of https://github.com/dmlc/mxnet/blob/master/example/notebooks/predict-with-pretrained-model.ipynb  get_internals allows you to get any internal symbols

**Comment by Yunrui** [comment]
Yes, it's what I am looking for. Thanks so much!

**Comment by piiswrong** [comment]
@tqchen Let's add using BlockGrad for this to some examples?
